### ISSUE TITLE
create a page / demo of how folks who just have maven can spin up a kubernetes cluster to be able to try `mvn fabric8:run` on their project

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8-spring#2]
e.g. via the new `mvn fabric8:install' command https://github.com/fabric8io/fabric8-maven-plugin/issues/299 to setup minikube / kubectl / gofabric8 stuff (hopefully we can hide the last one really)

**Comment by jstrachan** [comment]
i.e. so that with the combination of #1 and this issue, folks can take any old maven project, fabric8 enable it, create a local cluster and run it in a couple of commands without ever having to know what minikube or kubectl is

**Comment by rawlingsj** [comment]
yeah - using terminology that is familiar and helps with an easy introduction to developing java for the cloud
